api/src/paths/projects/{projectId}/artifacts/list.ts

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+import { RequestHandler } from 'express';
+import { Operation } from 'express-openapi';
+import { WRITE_ROLES } from '../../../../constants/roles';
+import { getFileListFromS3 } from '../../../../utils/file-utils';
+import { getLogger } from '../../../../utils/logger';
+
+const defaultLog = getLogger('/api/projects/{projectId}/artifacts/list');
+
+export const GET: Operation = [getMediaList()];
+
+GET.apiDoc = {
+  description: 'Fetches a list of artifact descriptions in a project.',
+  tags: ['artifacts'],
+  security: [
+    {
+      Bearer: WRITE_ROLES
+    }
+  ],
+  parameters: [
+    {
+      in: 'path',
+      name: 'projectId',
+      required: true
+    }
+  ],
+  responses: {
+    200: {
+      description: 'Project get response file description array.',
+      content: {
+        'application/json': {
+          schema: {
+            type: 'array',
+            items: {
+              type: 'object',
+              properties: {
+                fileName: {
+                  description: 'The file name of the artifact',
+                  type: 'string'
+                },
+                lastModified: {
+                  description: 'The date the object was last modified',
+                  type: 'string'
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    401: {
+      $ref: '#/components/responses/401'
+    },
+    503: {
+      $ref: '#/components/responses/503'
+    },
+    default: {
+      $ref: '#/components/responses/default'
+    }
+  }
+};
+
+function getMediaList(): RequestHandler {
+  return async (req, res) => {
+    defaultLog.debug('started... req.params.projectId=' + req.params.projectId);
```

Comment:
For the defaultLog calls, can you update this (and any others) to match the existing pattern:
```Typescript
defaultLog.debug({label: 'getMediaList', message: 'params', req.params);
```

Specifically, the `label` being set to the function name (or some other relevant label if there is no function).  The `message` being set to your log message.
You can keep the projectId as part of the message if you want, but you can also just throw the entire `req.params` object at the end, and it will get printed. Any number of objects can be added actually, and they will all get printed out.